doc/api/fs.md

```diff
@@ -7510,6 +7510,23 @@ added:
 
 Free blocks available to unprivileged users.
 
+```cjs
+const fs = require('node:fs');
+// Calculate available space in bytes
+fs.statfs('.', (err, stats) => {
```

Comment:
This isn't the same as the other example:
```suggestion
const { statfs } = require('node:fs');
// Calculate available space in bytes
statfs('.', (err, stats) => {
```

Reply:
> This isn't the same as the other example:

i have corected it now